gnomad_qc/v4/sample_qc/assign_ancestry.py

```diff
@@ -84,6 +138,8 @@ def prep_ht_for_rf(
     seed: int = 24,
     test: bool = False,
     only_train_on_hgdp_tgp: bool = False,
+    v4_population_spike: List[str] = None,
+    v3_population_spike: List[str] = None,
 ) -> hl.Table:
     """
     Prepare the PCA scores hail Table for the random forest population assignment runs.
```

Comment:
Might be good to add a bit more detail to this about the options, like instead of the comment on line 166, put that info here and expand it to include v3 and v4 options